### PR TITLE
Launch campaign

### PR DESCRIPTION
**Pull request by SamPeck**
Launch email campaign emails for Moodlerooms and Blackboard customers.

### CODE REVIEW

File: email/ally-fulfilment-inline.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                                                                    </td>
+                                                                </tr>
+                                                            </tbody>
+                                                        </table>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                    </td>
```
Comment (mrvisser) [inline email/ally-fulfilment-inline.html]: These empty cells could be an accessibility issue, but I have no idea at what point that actually matters in a table layout for email. A more semantic approach if we're just looking for empty space would be to use margin/padding.

File: email/ally-fulfilment-inline.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                                                                        <table border="0" cellpadding="0" cellspacing="0" class="btn btn-primary" style="border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;box-sizing:border-box;width:100%;text-align:center;padding:16px;text-align:left;">
+                                                                            <tbody>
+                                                                                <tr>
+                                                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                                                        <a href="#" title="Book a reporting run-through" target="_blank" style="color:#191F25;text-decoration:underline;background-color:#FFFFFF;border:solid 1px #007900;box-sizing:border-box;color:#007900;cursor:pointer;display:inline-block;font-size:15px;font-weight:bold;line-height:1.6;margin:0;padding:8px 16px;text-decoration:none;text-transform:capitalize;background-color:#44474F;border-color:#44474F;color:#FFFFFF;">Book now</a>
```
Comment (mrvisser) [inline email/ally-fulfilment-inline.html:186]: Out of curiosity, what is the value of a `#`-href'd link in an email?

File: email/ally-fulfilment-inline.html [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
```
Comment (mrvisser) [inline email/ally-fulfilment-inline.html:186]: Also, if it's actually performing an action to "Book now", perhaps it should be a `button` instead. If it's just taking them to a website to book though then a link is fine I think.

File: email/ally-fulfilment.html [hunk 4]
```diff
@@ -0,0 +1,639 @@
...
+                                                                    </td>
+                                                                </tr>
+                                                            </tbody>
+                                                        </table>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td>
+                                                    </td>
+                                                </tr>
```
Comment (mrvisser) [inline email/ally-fulfilment.html]: Can this row be removed?

File: email/ally-fulfilment-inline.html (hunk 2 above)
Comment (SamPeck) [reply email/ally-fulfilment-inline.html:186]: It's a placeholder for marketing people to stick in Eloqua code

File: email/ally-fulfilment-inline.html (hunk 3 above)
Comment (SamPeck) [reply email/ally-fulfilment-inline.html:186]: It's just taking them to a webpage with a form

File: email/bb-ally-launch.html [hunk 5]
```diff
@@ -0,0 +1,636 @@
...
+                        </table>
+                        <!-- END CENTERED WHITE CONTAINER -->
+                        <table border="0" cellpadding="0" cellspacing="0" class="footer">
+                            <tbody>
+                                <tr>
+                                    <td>
+                                        <img alt="Blackboard" border="0" height="15" src="_assets/img/blackboardLogo.png" width="107">
+                                        <p>1111 19th Street NW, 9th Floor Washington, DC 20036 USA</p>
+                                        <p>Esta mensaje fue enviado a <a href="mailto:Elisa.Viali@blackboard.com" title="Elisa Viali">Elisa.Viali@blackboard.com</a> por Blackboard</p>
+                                        <p>© 1997 - 2017 Blackboard Inc. Todos los derechos reservados.</p>
```
Comment (mrvisser) [inline email/bb-ally-launch.html]: This is spanish but the content of the email seems to be in english?

File: email/bb-ally-reminder.html [hunk 6]
```diff
@@ -0,0 +1,618 @@
...
+                        </table>
+                        <!-- END CENTERED WHITE CONTAINER -->
+                        <table border="0" cellpadding="0" cellspacing="0" class="footer">
+                            <tbody>
+                                <tr>
+                                    <td>
+                                        <img alt="Blackboard" border="0" height="15" src="_assets/img/blackboardLogo.png" width="107">
+                                        <p>1111 19th Street NW, 9th Floor Washington, DC 20036 USA</p>
+                                        <p>Esta mensaje fue enviado a <a href="mailto:Elisa.Viali@blackboard.com" title="Elisa Viali">Elisa.Viali@blackboard.com</a> por Blackboard</p>
+                                        <p>© 1997 - 2017 Blackboard Inc. Todos los derechos reservados.</p>
```
Comment (mrvisser) [inline email/bb-ally-reminder.html]: Spanish here too

File: email/moodlerooms-ally-launch.html [hunk 7]
```diff
@@ -0,0 +1,636 @@
...
+                        </table>
+                        <!-- END CENTERED WHITE CONTAINER -->
+                        <table border="0" cellpadding="0" cellspacing="0" class="footer">
+                            <tbody>
+                                <tr>
+                                    <td>
+                                        <img alt="Blackboard" border="0" height="15" src="_assets/img/blackboardLogo.png" width="107">
+                                        <p>1111 19th Street NW, 9th Floor Washington, DC 20036 USA</p>
+                                        <p>Esta mensaje fue enviado a <a href="mailto:Elisa.Viali@blackboard.com" title="Elisa Viali">Elisa.Viali@blackboard.com</a> por Blackboard</p>
+                                        <p>© 1997 - 2017 Blackboard Inc. Todos los derechos reservados.</p>
```
Comment (mrvisser) [inline email/moodlerooms-ally-launch.html]: Spanish

File: email/moodlerooms-ally-reminder.html [hunk 8]
```diff
@@ -0,0 +1,618 @@
...
+                                        <p>Esta mensaje fue enviado a <a href="mailto:Elisa.Viali@blackboard.com" title="Elisa Viali">Elisa.Viali@blackboard.com</a> por Blackboard</p>
+                                        <p>© 1997 - 2017 Blackboard Inc. Todos los derechos reservados.</p>
+                                    </td>
+                                    <td>
+                                        <ul>
+                                            <li><a href="#" title="Acerca de Blackboard">Acerca de Blackboard</a></li>
+                                            <li><a href="#" title="Contáctenos">Contáctenos</a></li>
+                                            <li><a href="#" title="Administre sus preferencias">Administre sus preferencias</a></li>
+                                            <li><a href="#" title="Cancelar subscripción">Cancelar subscripción</a></li>
+                                            <li><a href="#" title="Política de privacidad">Política de privacidad</a></li>
```
Comment (mrvisser) [inline email/moodlerooms-ally-reminder.html]: Spanish

**Pull request merged**